vfox-restServer/src/main/java/com/vfoexchange/restServer/controller/UserController.java

```diff
@@ -12,18 +15,24 @@
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
+
+import javax.imageio.ImageIO;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
 import java.util.List;
 
 @RestController
 public class UserController {
 
+    @Autowired
+    private Environment environment;
     @Autowired
     private UserService userService;
     @Autowired
     private EmailServices emailServices;
 
-    @Autowired
-    Environment environment;
 
     private static Logger LOGGER = LoggerFactory.getLogger(UserController.class);
     /*
```

Comment:
LOGGER should be the first line on all classes.

Reply:
Done !